src/Symfony/Component/DependencyInjection/Tests/EnvVarProcessorTest.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\EnvVarProcessor;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+
+class EnvVarProcessorTest extends TestCase
```

Comment:
would you mind opening a separate PR against branch 3.4 for this test class?

Reply:
Ok, I moved to #26542, but had to remove the case for `json:null` because it will fail in 3.4, and i cannot add it here until that PR lands in master